tsdb/mapper.go

```diff
@@ -216,7 +216,7 @@ func (lm *LocalMapper) Open() error {
 					// No data exists for this key.
 					continue
 				}
-				seriesTags := lm.shard.index.series[key].Tags
```

Comment:
no, it was a data race because it was accessing a shared map without a lock. The series is guaranteed to be in the map when you get to this point.